### PR TITLE
refactor(RHTAPREL-715): rename PipelineRun references

### PR DESCRIPTION
**Pull request by davidmogar**
Rename references to the release Pipeline{Run} to managed Pipeline{Run}.

### CODE REVIEW

**Review by mmalina — DISMISSED** [review]

**Comment by scoheb** [comment]
Can we consider `Release` -> `managed Release` and `tenant Release` pipelines?

**Comment by scoheb** [comment]
> > Can we consider `Release` -> `managed Release` and `tenant Release` pipelines?
> 
> In some cases I talk about Release Pipelines or Release PipelineRuns. That is, managed, tenant and post release Pipelines or PipelineRuns.
> 
> Does this answer your question?

I am referring to: https://github.com/redhat-appstudio/release-service/pull/310/files#r1410721270

File: Makefile [hunk 1]
```diff
@@ -8,10 +8,10 @@ TAG_NAME ?= next
 CERT_MANAGER_VERSION ?= v1.8.0
 ENABLE_WEBHOOKS ?= true
 
-# DEFAULT_PERSISTENT_VOLUME_CLAIM defines the default PVC to be used in the Release pipeline workspace declaration.
+# DEFAULT_PERSISTENT_VOLUME_CLAIM defines the default PVC to be used in the managed Pipeline workspace declaration.
```
[the remark this answers is not in the document]
Comment (davidmogar) [reply Makefile:11]: I don't call it `managed Release Pipeline` in any place. Do you want me to change this in every single mention?

[the remark this answers is not in the document]
Comment (scoheb) [reply Makefile:11]: I think we should say Managed Release Pipeline and Tenant Release pipeline. 

Just saying managed pipeline is not enough in my opinion

File: controllers/release/adapter.go [hunk 2]
```diff
@@ -331,12 +331,12 @@ func (a *adapter) EnsureReleaseProcessingIsTracked() (controller.OperationResult
...
 
-// createReleasePipelineRun creates and returns a new release PipelineRun. The new PipelineRun will include owner
+// createManagedPipelineRun creates and returns a new managed Release PipelineRun. The new PipelineRun will include owner
 // annotations, so it triggers Release reconciles whenever it changes. The Pipeline information and the parameters to it
 // will be extracted from the given ReleaseStrategy. The Release's Snapshot will also be passed to the release
 // PipelineRun.
-func (a *adapter) createReleasePipelineRun(resources *loader.ProcessingResources) (*tektonv1.PipelineRun, error) {
-	pipelineRun := tekton.NewReleasePipelineRun("release-pipelinerun", resources.ReleasePlanAdmission.Namespace).
+func (a *adapter) createManagedPipelineRun(resources *loader.ProcessingResources) (*tektonv1.PipelineRun, error) {
+	pipelineRun := tekton.NewReleasePipelineRun("managed-pipelinerun", resources.ReleasePlanAdmission.Namespace).
```
[the remark this answers is not in the document]
Comment (davidmogar) [reply controllers/release/adapter.go:339]: WDYT about using `managed-release` only and remove the `pipelinerun` part so it's a bit sorter? It's a bit redundant when you are already searching for PipelineRuns.

[the remark this answers is not in the document]
Comment (johnbieren) [reply controllers/release/adapter.go:339]: Sounds great to me

**Review by johnbieren — APPROVED** [review]

**Pull request merged**